src/raft/uv_metadata.c

```diff
@@ -26,12 +26,7 @@ static int uvMetadataDecode(const void *buf,
 			    char *errmsg)
 {
 	const void *cursor = buf;
-	uint64_t format;
-	format = byteGet64(&cursor);
-	if (format != UV__DISK_FORMAT) {
-		ErrMsgPrintf(errmsg, "bad format version %ju", format);
-		return RAFT_MALFORMED;
-	}
+	metadata->format_version = byteGet64(&cursor);
```

Comment:
Do we need to validate here that `0 < format_version < UV_FORMAT_NR`?

Reply:
Good catch, I'd been thinking that that this was covered by the check in raft_uv_set_format_version plus the comparison of `uv->format_version` and `metadata.format_version`---but if you don't call raft_uv_set_format version then the first one will never happen. So indeed I should restore the previous check here.